sites/partners/src/listings/Aside.tsx

```diff
@@ -91,6 +91,7 @@ const Aside = ({
               fullWidth
               onClick={() => false}
               type="button"
+              dataTestId="listingEditButton"
```

Comment:
In other places we use `data-test-id`, so I think it would be great to follow this pattern.

Reply:
Yep we're doing that when we need a custom test id for components that have multiple of them on the page

Reply:
my bad! I thought it's not a prop, looks good to me :)